src/main/app/forms/models/defendantDetails.ts

```diff
@@ -1,48 +1,48 @@
 import { Serializable } from 'app/models/serializable'
 import { IsDefined, IsIn, IsOptional, MaxLength, ValidateIf } from 'class-validator'
-import { DefendantTypes } from 'app/forms/models/defendantTypes'
+import { IndividualTypes } from 'app/forms/models/individualTypes'
 import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
 
 export class ValidationErrors {
-  static readonly DEFENDANT_TYPE_REQUIRED: string = 'Choose defendant type'
-  static readonly TITLE_REQUIRED: string = 'Enter defendant title'
-  static readonly FULLNAME_REQUIRED: string = 'Enter full name'
-  static readonly ORGANISATION_NAME_REQUIRED: string = 'Enter organisation name'
-  static readonly COMPANY_HOUSE_NUMBER_REQUIRED: string = 'Enter company house number'
+  static readonly DEFENDANT_TYPE_REQUIRED: string = 'Choose a type of defendant'
+  static readonly TITLE_REQUIRED: string = 'Enter a title'
+  static readonly FULLNAME_REQUIRED: string = 'Enter a full name'
+  static readonly ORGANISATION_NAME_REQUIRED: string = 'Enter an organisation name'
+  static readonly COMPANY_HOUSE_NUMBER_REQUIRED: string = 'Enter a valid Companies House number'
   static readonly CONTENT_TOO_LONG: string = 'You’ve entered too many characters'
 }
 
 export class DefendantDetails implements Serializable<DefendantDetails> {
 
   @IsDefined({ message: ValidationErrors.DEFENDANT_TYPE_REQUIRED })
-  @IsIn(DefendantTypes.all(), { message: ValidationErrors.DEFENDANT_TYPE_REQUIRED })
-  type?: DefendantTypes
+  @IsIn(IndividualTypes.all(), { message: ValidationErrors.DEFENDANT_TYPE_REQUIRED })
+  type?: IndividualTypes
 
-  @ValidateIf(o => o.type === DefendantTypes.INDIVIDUAL)
+  @ValidateIf(o => o.type === IndividualTypes.INDIVIDUAL)
```

Comment:
`IndividualTypes.INDIVIDUAL` does not look a better name. `IndividualTypes.ORGANISATION` also looks confusing.